lib/content/index.js

```diff
@@ -155,7 +155,9 @@ module.exports = {
     '/LICENSE*',
     '/CHANGELOG*',
   ],
-  ignorePaths: [],
+  ignorePaths: [
```

Comment:
I'll let @lukekarrys weigh in here since it's surprising to me coming in fresh on this that we don't have anything in here already.

Reply:
@rotu the current pattern is to put these paths in `lib/content/gitignore` and leave `ignorePaths` as a config to be set by consumers. can you make that change? apologies that this is not discoverable! 🙇🏼

Reply:
No worries. I can add the breadcrumbs that would have retroactively helped me. The `lib/content/gitignore` file is clearly a template of some sort. What instantiates this template?

Reply:
Also, any reason why `allowPaths` and `distPaths` are inlined in this file but `ignorePaths` is not?